resources/build/build-utils.sh

```diff
@@ -162,7 +162,7 @@ function exportEnvironmentDefinitionScript() {
     echo "export SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN:-}" >> "$ENVIRONMENT_SH"
 
     # Ensure the autogenerated file may be successfully run/included by xcode-utils.sh.
-    chmod a+x "$ENVIRONMENT_SH"
+    chmod +x "$ENVIRONMENT_SH"
```

Comment:
Turns out that macOS Catalina suddenly was giving prompts on any command-line build suggesting the parameter scheme was a touch off.  This minor alteration fixes that.